Server/Server.Core/Server.Core.Static/Settings.hpp

```diff
@@ -1,52 +1,31 @@
 #pragma once
 
-#include "ArgParser.hpp"
-
-#include <PostgreRepositoryManager.hpp>
+namespace Server::Builder
+{
+struct ParamType
+{
+    constexpr static char* ServerPort  = "--serverport";
```

Comment:
https://godbolt.org/z/6bbxYn59c
Need const

Reply:
Thanks for your attention. this's my fault.